conda/environment-dev.yaml

```diff
@@ -0,0 +1,11 @@
+name: nplinker-dev
```

Comment:
Please remove this conda env file. 
If you like conda, use it only to create virtual env, and don't use conda packages (that's also what I'm doing). Let's install all packages from pip and the `install-nplinker-deps` command. The reason is that at the moment we don't want to solve conda conflicts for installing nplinker and its dependencies.

To create an dev env:
```
conda create -y --name nplinker-dev python=3.9
pip install -e .
install-nplinker-deps
```
I'll update README to make creating dev env clear.